### ISSUE TITLE
Aa a resident I want to be able to sort list of hearings to any order I see fit.

### ISSUE THREAD

**Issue by itimo** [City-of-Helsinki/kerrokantasi-ui#13]

**Comment by itimo** [comment]
Possible orders:
- newest first
- oldest first
- alphabetical order (and reverse)
- most commented (and least)
- according to closing time

**Comment by Rikuoja** [comment]
I think perhaps open/closed/all hearings should be a separate selection, in addition to sorting?

Current service has newest first ordering, but it buries some open hearings far away below already closed ones.

**Comment by itimo** [comment]
Good point @Rikuoja . I will add additional use case for filtering lists.
